### PR TITLE
Improve spec_helper to report coverage

### PR DESCRIPTION
**Pull request by felipsimoes**
This PR: 
- removes a required byebug unused and not imported on the gemfile dependencies.
- includes simplecov on development dependencies
- require simplecov on spec_helper for coverage report

### CODE REVIEW

**Comment by arturcp** [comment]
Thank you very much for your contribution, @felipsimoes 👏 The test coverage is very important, and I liked the small improvements you did as well, like ordering the dependencies 👍

File: spec/tools/database_spec.rb [hunk 1]
```diff
@@ -1,3 +1,4 @@
+require_relative '../spec_helper'
```
Comment (arturcp) [inline spec/tools/database_spec.rb:1]: I believe this line is unnecessary. The tests run well without it. Can you double check?

File: spec/configuration_spec.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+require 'spec_helper'
```
Comment (arturcp) [inline spec/configuration_spec.rb:1]: I believe this line is unnecessary. The tests run well without it. Can you double check?

File: lib/tools/database.rb [hunk 3]
```diff
@@ -1,4 +1,3 @@
-require 'byebug'
```
Comment (arturcp) [inline lib/tools/database.rb:1]: 👏

File: spec/configuration_spec.rb (hunk 2 above)
Comment (felipsimoes) [reply spec/configuration_spec.rb:1]: Sure, I will check both require statements.

File: spec/tools/database_spec.rb (hunk 1 above)
Comment (felipsimoes) [reply spec/tools/database_spec.rb:1]: This can really be removed with no side effects.

File: spec/configuration_spec.rb (hunk 2 above)
Comment (felipsimoes) [reply spec/configuration_spec.rb:1]: Once I remove this spec_helper the configuration file is not considered on the coverage report, maybe due to different folder hierarchy. 🤔

**Comment by arturcp** [comment]
@felipsimoes I found the solution. Remove all `require 'spec_helper'` from all files and add this to the `.rspec`, in the root folder:

```
--require spec_helper
```

File: .rspec [hunk 4]
```diff
@@ -0,0 +1 @@
+--require spec_helper
```
Comment (arturcp) [inline .rspec:1]: Can you add two more lines on this file? 

```
--color
--format documentation
--require spec_helper

```

**Review by arturcp — APPROVED** [review]

File: .rspec (hunk 4 above)
Comment (felipsimoes) [reply .rspec:3]: Done 👍  thanks

**Pull request merged**